FSQLocationBroker/FSQLocationBroker.h

```diff
@@ -532,3 +518,5 @@ typedef NS_OPTIONS(NSUInteger, FSQLocationSubscriberOptions) {
 - (void)monitoringDidFailForRegion:(CLRegion *)region withError:(NSError *)error;
```

Comment:
error might want to be nullable?

Reply:
I think error will always be present since this is just a wrapper for locationManager(_:monitoringDidFailFor:withError:). With that being the case though region should be `nullable`.

 https://developer.apple.com/documentation/corelocation/cllocationmanagerdelegate/1423720-locationmanager

Reply:
=bueno, this is following Apple's prototype for this protocol

Reply:
@mdyson Just want to make sure you catch this as well before merging in so we are in sync with apple's delegate method as well with region being `nullable`

Reply:
fixed!